### DISCUSSION TITLE
GitHub stats

### DISCUSSION THREAD

**Discussion by mrafaellb** [community/community#43147] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello guys, does anyone know how I can show my GitHub stats on the first page of my profile?

**Comment by artie93** [comment, marked as answer]
Hi
yeah, that's easy, just insert this in your own readme.md file of your profile: https://github-readme-stats.vercel.app/api?username=your_username&show_icons=true&theme=transparent&include_all_commits=true&count_private=true
you can also edit its size using markdown. Check out my profile if you want to see how it works.

Hope this helps,
Best regards

**Reply by mrafaellb** [reply]
Awesome, I'll try it out. Thank you!
Btw, how do I resize this gif on your profile?

**Reply by artie93** [reply]
No problem
you can try to play around with the values of width and height. For instance:
< img height="180em" width="400em" src="https://github-readme-stats.vercel.app/api?username=artie93&show_icons=true&theme=transparent&include_all_commits=true&count_private=true"/>
(remove the first blank space in order to see the result)
Hope this is helpful
